scripts/core/context.yml

```diff
@@ -182,6 +182,19 @@ params:
       desc: |
             [out] a pointer to the native handle of the context.
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Descriptor for $xContextCreateWithNativeHandle."
+class: $xContext
+name: $x_context_native_desc_t
```

Comment:
In UR these creation structs have been using the `*_properties_t` naming rather than `*_desc_t`. Its crossed my mind a few times if that was not the intention of the scripts but for consistency I think this should be `*_properties_t`.

Reply:
Good idea, Changing.